src/v/pandaproxy/test/fetch.cc

```diff
@@ -104,7 +104,7 @@ FIXTURE_TEST(pandaproxy_fetch, pandaproxy_test_fixture) {
           res.headers.result(), boost::beast::http::status::ok);
         BOOST_REQUIRE_EQUAL(
           res.body,
-          R"([{"topic":"t","key":"","value":"dmVjdG9yaXplZA==","partition":0,"offset":1},{"topic":"t","key":"","value":"cGFuZGFwcm94eQ==","partition":0,"offset":2},{"topic":"t","key":"","value":"bXVsdGlicm9rZXI=","partition":0,"offset":3}])");
+          R"([{"topic":"t","key":"AAD//w==","value":"","partition":0,"offset":0},{"topic":"t","key":"","value":"dmVjdG9yaXplZA==","partition":0,"offset":1},{"topic":"t","key":"","value":"cGFuZGFwcm94eQ==","partition":0,"offset":2},{"topic":"t","key":"","value":"bXVsdGlicm9rZXI=","partition":0,"offset":3}])");
```

Comment:
This is fine if the output is binary JSON (base64 encoded), but I wonder what happens when the output is JSON.

This is probably a rhetorical question - if it should be returned to the client, I'll have to decode it as something like:
```json
{"key": "control_<offset>", "value": "", "partition": 0, "offset": 0}
```

Reply:
The control record key is a tuple of `version` and `control_record_type`. I would go with: 
```json
{
	"key": {
		"version": 0,
		"type": -1
	},
	"value": "",
	"partition": 0,
	"offset": 0
}
```

Reply:
There might be problems with the schema, I'll have to figure out how it works.